integration_tests/test_ibc_rly.py

```diff
@@ -46,6 +46,18 @@ def ibc(request, tmp_path_factory):
     "prepare-network"
     path = tmp_path_factory.mktemp("ibc_rly")
     procs = []
+
+    def kill_procs():
+        for proc in procs:
+            try:
+                os.killpg(os.getpgid(proc.pid), signal.SIGTERM)
+            except ProcessLookupError:
+                pass
+            # proc.terminate()
+            proc.wait()
+
+    signal.signal(signal.SIGINT, kill_procs)
```

Comment:
@yihuang Not quite familiar with python, not sure if you have better idea

Reply:
Manage them in supervisor?

Reply:
It's using already, but pid was not return correctly

Reply:
supervisord can't stop it properly?

Reply:
maybe since rly hasnt use supervisor yet and currently it's generic to kill all proccesses pid for prepare networks.

Reply:
I had to kill supervisord manually when the integration test fails otherwise it keeps restarting the procs

Reply:
> I had to kill supervisord manually when the integration test fails otherwise it keeps restarting the procs

should have fixed now.